include/subnet_cadmium.hpp

```diff
@@ -67,8 +68,8 @@ class Subnet {
 
         // external transition
         void external_transition(
-		            TIME e,
-			    typename make_message_bags<input_ports>::type mbs) { 
+	    TIME e,
+	        typename make_message_bags<input_ports>::type mbs) { 
             state.index ++;
```

Comment:
No space is required before increment operator (++).
i.e., state.index++;

Reply:
Resolved.